src/helpers/queryHelper.ts

```diff
@@ -0,0 +1,30 @@
+import { consts, texts } from '../config';
+import { QUERY_TYPES } from '../queries';
+
+const { ROOT_ROUTE_NAMES } = consts;
+
+/**
+ * Determines the root route name for data based on its query.
+ */
+export const rootRouteName = (query: string) =>
+  ({
+    [QUERY_TYPES.EVENT_RECORD]: ROOT_ROUTE_NAMES.EVENT_RECORDS,
+    [QUERY_TYPES.NEWS_ITEM]: ROOT_ROUTE_NAMES.NEWS_ITEMS,
+    [QUERY_TYPES.POINT_OF_INTEREST]: ROOT_ROUTE_NAMES.POINTS_OF_INTEREST_AND_TOURS,
+    [QUERY_TYPES.TOUR]: ROOT_ROUTE_NAMES.POINTS_OF_INTEREST_AND_TOURS
+  }[query]);
+
+export const getTitleForQuery = (query: string) => {
+  switch (query) {
+    case QUERY_TYPES.NEWS_ITEMS:
+      return texts.homeCategoriesNews.categoryTitle;
```

Comment:
the text for news items is a fallback if there is none from global settings, right?
so it can differ here if there are different texts existing in global settings, right?

Reply:
that is correct. however this was necessary due to us ignoring news categories there.
i added categories for news now.